### DISCUSSION TITLE
when upload a folder to B2C commerce site I got errors

### DISCUSSION THREAD

**Discussion by Flora-Java** [iterate-ch/cyberduck#14819] [Q&A]
I am trying to upload a folder with subfolders to Salesforce B2C commerce. I know it works because somebody on my team is using it, but she is not available until Monday. After I upload the file, I am getting 403 Forbidden error. I am attaching a screenshot. I hope you can help me
![Screenshot 2023-06-16 at 1 46 00 PM](https://github.com/iterate-ch/cyberduck/assets/48796428/323fcc56-fccf-4de4-b7bd-6ae4a0cfc32c)

**Comment by tatze96** [comment]
Please verify that your user account has write permissions for the affected folder.

If you have the corresponding write permissions, [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the mentioned error, and send us the log file afterward. 

If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14819 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
